api/src/pcapi/routes/backoffice/users/blueprint.py

```diff
@@ -230,3 +235,36 @@ def confirm_batch_suspend_users() -> utils.BackofficeResponse:
         flash(f"{len(users)} compte d'utilisateur a été suspendu", "success")
 
     return redirect(url_for("backoffice_web.fraud.list_blacklisted_domain_names"), code=303)
+
+
+@users_blueprint.route("/<int:user_id>/redirect_to_brevo", methods=["GET"])
+@utils.permission_required_in(
+    [
+        perm_models.Permissions.READ_PUBLIC_ACCOUNT,
+        perm_models.Permissions.READ_PRO_ENTITY,
+    ]
+)
+def redirect_to_brevo_user_page(user_id: int) -> utils.BackofficeResponse:
+    user = (
+        users_models.User.query.filter_by(id=user_id).options(joinedload(users_models.User.UserOfferers)).one_or_none()
```

Comment:
Je me demandais pourquoi tu faisais la jointure sur `UserOfferers`. Je vois dans le code que c'est par rapport à la condition dans `_redirect_to_user_page`. Cependant, depuis la création du rôle `NON_ATTACHED_PRO`, nous avons changé la condition, mais oubliée ici. C'est donc l'occasion de remplacer : 
```
def _redirect_to_user_page(user: users_models.User) -> utils.BackofficeResponse:
    # Actions should always come from user details page
    if request.referrer:
        return redirect(request.referrer, code=303)

    # Fallback in case referrer is missing (maybe because of browser settings)
    if user.UserOfferers:
        return redirect(url_for("backoffice_web.pro_user.get", user_id=user.id), code=303)

    return redirect(url_for("backoffice_web.public_accounts.get_public_account", user_id=user.id), code=303)
```
par : 
```
def _redirect_to_user_page(user: users_models.User) -> utils.BackofficeResponse:
    # Actions should always come from user details page
    if request.referrer:
        return redirect(request.referrer, code=303)

    # Fallback in case referrer is missing (maybe because of browser settings)
    if user.has_pro_role or user.has_non_attached_pro_role:
        return redirect(url_for("backoffice_web.pro_user.get", user_id=user.id), code=303)

    return redirect(url_for("backoffice_web.public_accounts.get_public_account", user_id=user.id), code=303)
```
et donc par cette occasion la jointure là où elle n'est plus nécessaire.